invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/PatronDetails/components/ItemsCheckout/state/actions.js

```diff
@@ -13,13 +13,18 @@ export const checkoutItem = (item, patronPid, shouldForceCheckout = false) => {
       type: IS_LOADING,
     });
     const currentUser = sessionManager.user;
+    const loan = {
+      metadata: {
+        item_pid: item.item_pid,
+        patron_pid: String(patronPid),
```

Comment:
the patron pid should be serialized to string in the serializer (in /api/user.js), as it comes as a number but everywhere we use it as the string